lib/src/compiler.dart

```diff
@@ -36,6 +36,7 @@ import 'info.dart'
     show AnalyzerMessage, CheckerResults, LibraryInfo, LibraryUnit;
 import 'options.dart';
 import 'report.dart';
+import 'package:dev_compiler/src/server/server.dart';
```

Comment:
trivial, but this should probably be "src/server/server.dart" to match the style of the other imports